transformations/dev-prod-mode/dev-prod-mode.md

```diff
@@ -0,0 +1,285 @@
+f.write("{{ outFileContent }}")
+f.close()
+
+print("Output written to {{ outFile }}")
+{% endhighlight %}
+
+{: .image-popup}
+![Screenshot - Edit Shared Code](/transformations/dev-prod-mode/13-edit-shared-code-in-dev.png)
+
+The transformation will do exactly the same as before, but it will add `Output written to demoFile.txt` to 
+the output. Save the shared code. Go back to **Transformations**, select the transformation *Testing dev/prod 
+mode*, and run the component again.
+
+You can see in the job log that you used this new block.
+
+{: .image-popup}
+![Screenshot - Job Log](/transformations/dev-prod-mode/14-jobs-log.png)
+
+The file was uploaded to Storage again. 
+
+Now change the name of the output file. To do that, go to your transformation and change the value of the variable 
+`outFile` from `demoFile.txt` to `demoBranchFile.txt`. Then click the little check mark icon on the right.
+
+{: .image-popup}
+![Screenshot - Edit Variable in Development Branch](/transformations/dev-prod-mode/15-edit-var-in-dev.png)
+
+### Show Project Diff
+Go to the dashboard and click the button **SHOW PROJECT DIFF** on the right. There you can see the things you’ve 
+changed (the shared code and the definition of one variable for the Python transformation). 
+
+{: .image-popup}
+![Screenshot - Project Diff](/transformations/dev-prod-mode/16-project-diff.png)
+
+To see a detailed diff of the changes, click the three dots on the right and then **Compare with production**.
+
+{: .image-popup}
+![Screenshot - Config Changes - Shared Code](/transformations/dev-prod-mode/17-diff-config-changes-shared-code.png)
+
+{: .image-popup}
+![Screenshot - Config Changes - Variable](/transformations/dev-prod-mode/18-diff-config-changes-var.png)
+
+## Switch Back to Production
+Let’s go back to production to see that by changing the transformation in the development branch, we did not 
+affect the production transformation in any way. 
+
+Go to **Transformations** and check **Shared Codes** to see that there is still the original version of 
+`Create a file` that we created at the beginning. It was not overwritten by the changes we made in the development 
+branch.
+
+{: .image-popup}
+![Screenshot - Check Shared Code In Production](/transformations/dev-prod-mode/19-check-shared-code.png)
+
+Then go to your production transformation and check **Block 1** to see that there, too, is the original short 
+version of the shared code. 
+
+{: .image-popup}
+![Screenshot - Check Variable In Production](/transformations/dev-prod-mode/20-check-block1.png)
+
+To make sure nothing has changed, run the transformation in production again. When it finishes, go to **Storage -- 
+Files**. You can see that `demoFile.txt` was created. Because it ran in production mode, it was assigned the 
+tag `demoOutput` without any prefix.
+
+{: .image-popup}
+![Screenshot - Storage File in Production](/transformations/dev-prod-mode/21-storage-files-prod.png)
+
+##  Simultaneous Changes
```

Comment:
toto bych hodil až za /transformations/dev-prod-mode/#merge-to-production
je to zase krok stranou